systemtest/src/test/java/io/strimzi/systemtest/report/DefaultClusterReportST.java

```diff
@@ -0,0 +1,453 @@
+
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        Exec.exec(USER_PATH + "/../tools/report.sh",
+            "--namespace=" + testStorage.getNamespaceName(),
+            "--cluster=" + testStorage.getClusterName(),
+            "--out-dir=" + outPath,
+            "--secrets=off");
+
+        assertThat("Secrets are reported", !FileUtils.fileExists(secretPath));
+    }
+
+    @ParallelTest
+    void createReportWithoutNsOpts() {
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        assertThrows(KubeClusterException.class, () -> Exec.exec(USER_PATH + "/../tools/report.sh", "--cluster=" + testStorage.getClusterName()));
+    }
+
+    @ParallelTest
+    void createReportWithoutClusterOpts() {
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        assertThrows(KubeClusterException.class, () -> Exec.exec(USER_PATH + "/../tools/report.sh", "--namespace=" + testStorage.getNamespaceName()));
+    }
+
+    @ParallelTest
+    void createReportWithNonExistingNs() {
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        assertThrows(KubeClusterException.class, () -> Exec.exec(USER_PATH + "/../tools/report.sh",
+            "--namespace=non-existing-ns", "--cluster=" + testStorage.getClusterName()));
+    }
+
+    @ParallelTest
+    void createReportWithNonExistingCluster() {
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        assertThrows(KubeClusterException.class, () -> Exec.exec(USER_PATH + "/../tools/report.sh",
+            "--namespace=" + testStorage.getNamespaceName(), "--cluster=non-existing-k"));
+    }
+
+    @ParallelTest
+    void createReportWithUnknownSecretVerbosity() {
+        LOGGER.info("Running report on {}/{}", testStorage.getNamespaceName(), testStorage.getClusterName());
+        assertThrows(KubeClusterException.class, () -> Exec.exec(USER_PATH + "/../tools/report.sh",
+            "--namespace=" + testStorage.getNamespaceName(), "--cluster=" + testStorage.getClusterName(), "--secrets=unknown"));
+    }
+
+    @BeforeAll
+    void setUp(final ExtensionContext extensionContext) {
+        this.clusterOperator = this.clusterOperator
+            .defaultInstallation(extensionContext)
+            .withReplicas(2)
+            .createInstallation()
+            .runInstallation();
+        this.testStorage = new TestStorage(extensionContext, Constants.CO_NAMESPACE);
+        resourceManager.createResourceWithWait(extensionContext,
+            KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build(),
+            KafkaTemplates.kafkaEphemeral(testStorage.getClusterName() + "-tgt", 1).build()
```

Comment:
You can create class variable for the target cluster and here you can assign value to it, then use it in all other methods, or?